app/views/decidim/vocdoni/admin/census/index.html.erb

```diff
@@ -1,65 +1,82 @@
-<% add_decidim_page_title(translated_attribute(election.title)) %>
 <% add_decidim_page_title(t(".title")) %>
```

Comment:
Can we leave the resource title in the page title please? So it's easier to differentiate from other resources.  Also let's add a break-line (as it was before) so it easy to have the meta and body blocks visually separated. 

```suggestion
<% add_decidim_page_title(translated_attribute(election.title)) %>
<% add_decidim_page_title(t(".title")) %>

```